site/src/component/GradeDist/Pie.tsx

```diff
@@ -165,24 +177,11 @@ export default class Pie extends React.Component<PieProps> {
           enableArcLinkLabels={false}
           innerRadius={0.8}
           padAngle={2}
-          colors={['#60A3D1', '#81C284', '#F5D77F', '#ECAD6D', '#E8966D', '#4AB486', '#E36436']}
```

Comment:
It looks like the last two removed colors here are actually different from the other list. I think we should use the green and red for pass + no pass instead of making them gray